### ISSUE TITLE
Running containers die when machine suspended

### ISSUE THREAD

**Issue by v4tech** [docker/for-mac#85]
### Expected behavior

When I suspend my machine, running containers continue at wake.
### Actual behavior

When I suspend my machine, any running containers die with the error: 

> Error response from daemon: dial unix /Users/ME/Library/Containers/com.docker.docker/Data/*00000003.00000948: connect: connection refused

This is an issue that I raised 3 months ago...   12-May:
   https://forums.docker.com/t/running-containers-die-when-machine-suspended/11976
### Information

Diagnostic ID: 87389479-4F8F-4632-8BE8-591E5125A816
Docker for Mac: 1.12.0 (Build 10871)
macOS: Version 10.10.5 (Build 14F1909)
[OK]     docker-cli
[OK]     app
[OK]     moby-syslog
[OK]     disk
[OK]     virtualization
[OK]     system
[OK]     menubar
[OK]     osxfs
[OK]     db
[OK]     slirp
[OK]     moby-console
[OK]     logs
[OK]     vmnetd
[OK]     env
[OK]     moby
[OK]     driver.amd64-linux
### Steps to reproduce
1. Start a container
2. Close lid of laptop to suspend
3. Open lid and log in
4. Get annoyed that your containers are all dead and you have to restart them...

**Comment by sleep1661** [comment]
when will fix this?

**Comment by robshocks** [comment]
Experiencing this problem on Macbook pro yosemite 10.10.5 when Mac goes to sleep. Super frustrating to start containers all over again. Anyone have a fix or workaround?

**Comment by kodeninja** [comment]
+1

I have the same issue on my `MacBook Pro (Retina, 15-inch, Mid 2015)/ OS X Yosemite 10.10.5`.

Sometimes, I can just click on `restart` from the Docker icon from the dock, and it starts working properly. But mostly, I have to Quit it and relaunch docker.

```
Docker for Mac: version: mac-v1.12.0-beta21.1
OS X: version 10.10.5 (build: 14F1605)
logs: /tmp/E0CDD283-3799-4AC1-8200-473F5EDAF115/20160804-114325.tar.gz
[OK]     docker-cli
[OK]     app
[OK]     moby-syslog
[OK]     disk
[OK]     virtualization
[OK]     system
[OK]     menubar
[OK]     osxfs
[OK]     db
[OK]     slirp
[OK]     moby-console
[OK]     logs
[OK]     vmnetd
[OK]     env
[OK]     moby
[OK]     driver.amd64-linux
```

**Comment by sambernard** [comment]
I'm experiencing this as well, multiple times per hour after the lastest update. 

I do not have to suspend my computer for it to occur.

```
Diagnostic ID: ABC53D7A-8C28-42F0-BB36-0CF429E11C37
Docker for Mac: 1.12.0-beta21 (Build 11019)
macOS: Version 10.11.5 (Build 15F34)
[OK]     docker-cli
[OK]     app
[OK]     moby-syslog
[OK]     disk
[OK]     virtualization
[OK]     system
[OK]     menubar
[OK]     osxfs
[OK]     db
[OK]     slirp
[OK]     moby-console
[OK]     logs
[OK]     vmnetd
[OK]     env
[OK]     moby
[OK]     driver.amd64-linux
```

**Comment by v4tech** [comment]
Re a fix:  **Don't use native Docker**.

`docker-machine` works perfectly through sleep-wake cycles.

The only think that seems to cause it problems is when my machine runs low on power (~5%) and something (OSX?) causes it to die/kills it.

**Comment by dave-tucker** [comment]
For those who are experiencing this issue, please could you post the output of `sudo pmset -g` in addition to your Diagnose output. It would also be good to know if you have manually "tweaked" any of your Mac's power management settings (e.g I have mine set to disable hibernation). Using the latest version, I'm unable to replicate. I have an `nginx` container that's been running in the background for the last 3 hours while I've opened and closed my laptop lid on at least 10 occasions.

**Comment by amilstead** [comment]
👍 I have this issue as well, containers that were up when I close my lid are dead when I open it and log back in.

@v4tech I'm not positive that "Don't use native Docker" is really a great solution here...

On simple suspend/awake after a short period of time I don't need to restart but if it's been a while or I switch networks (e.g. VPN previously connected when lid was open and dies on lid close) I'll have to Quit the app and reload.

Diagnostic Info:

    Docker for Mac: version: win-beta-v1.12.0-rc5-beta21-73-g9cd1dc2
    OS X: version 10.10.5 (build: 14F27)
    logs: /tmp/F0FF0465-B609-4ED4-B336-20A2F37B3F63/20160805-104933.tar.gz
    [OK]     docker-cli
    [OK]     app
    [OK]     moby-syslog
    [OK]     disk
    [OK]     virtualization
    [OK]     system
    [OK]     menubar
    [OK]     osxfs
    [OK]     db
    [OK]     slirp
    [OK]     moby-console
    [OK]     logs
    [OK]     vmnetd
    [OK]     env
    [OK]     moby
    [OK]     driver.amd64-linux

For `sudo pmset -g`:

    $ sudo pmset -g
    Active Profiles:
    Battery Power        -1*
    AC Power        -1
    Currently in use:
     standbydelay         10800
     standby              1
     halfdim              1
     hibernatefile        /var/vm/sleepimage
     darkwakes            0
     gpuswitch            2
     disksleep            10
     sleep                0
     autopoweroffdelay    14400
     hibernatemode        3
     autopoweroff         1
     ttyskeepawake        1
     displaysleep         30
     acwake               0
     lidwake              1

**Comment by dave-tucker** [comment]
Thanks @amilstead 

> On simple suspend/awake after a short period of time I don't need to restart but if it's been a while or I switch networks (e.g. VPN previously connected when lid was open and dies on lid close)

This is *really* helpful. I have a theory about why this could be happening... It's possible this is related to OSX's "Safe Sleep" mode, which by the config above would happen 3hrs after your machine has been put in to Sleep.

For others responding to this, please could you let us know "how long" it's been after closing the lid that you notice this issue - hopefully we'll start seeing some correlation between that and the `standbydelay`

And if you know anyone who, like me has done a `sudo pmset -a hibernatemode 0`, it would be good to confirm if they see this issue at all.

As for the changing networks, this is know issue - I'm working on pushing a notification after a network change is detected to ask you if you want to restart.

**Comment by amilstead** [comment]
Awesome -- glad I could help! Should I try running `sudo pmset -a hibernatemode 0` and see if that prevents this going forward, or keep it non-zero for now?

**Comment by dave-tucker** [comment]
@amilstead that's up to you... 
From the manpage:

```
 hibernatemode = 0 (binary 0000) by default on supported desktops. The system will not back memory up to
     persistent storage. The system must wake from the contents of memory; the system will lose context on
     power loss. This is, historically, plain old sleep.
```

I wouldn't recommend this as a fix to anyone unless you are fully understand and are happy with the consequences 😀

**Comment by amilstead** [comment]
Got it -- thanks!

**Comment by Taik** [comment]
I'm also having this problem. Notably, I'm running under a modified hibernate mode  (25) for better battery life.

Diagnostic info:
```
Docker for Mac: version: mac-v1.12.0-beta21.1
OS X: version 10.11.6 (build: 15G31)
logs: /tmp/7C7664CF-B5E6-4AF2-8FD9-86FDDAD97A11/20160805-144645.tar.gz
[OK]     docker-cli
[OK]     app
[OK]     moby-syslog
[OK]     disk
[OK]     virtualization
[OK]     system
[OK]     menubar
[OK]     osxfs
[OK]     db
[OK]     slirp
[OK]     moby-console
[OK]     logs
[OK]     vmnetd
[OK]     env
[OK]     moby
[OK]     driver.amd64-linux
```

pmset:
```
$ sudo pmset -g
Password:
Active Profiles:
Battery Power		-1
AC Power		-1*
Currently in use:
 standbydelay         10800
 standby              1
 womp                 1
 halfdim              1
 hibernatefile        /var/vm/sleepimage
 powernap             1
 networkoversleep     0
 disksleep            0
 sleep                0
 autopoweroffdelay    14400
 hibernatemode        25
 autopoweroff         1
 ttyskeepawake        1
 displaysleep         30
 acwake               0
 lidwake              0
```

**Comment by sambernard** [comment]
_Deleted my previous comment and reposting with pmset data._

I'm experiencing this as well, multiple times per hour after the lastest update.

I do not have to suspend my computer for it to occur.

```
Diagnostic ID: ABC53D7A-8C28-42F0-BB36-0CF429E11C37
Docker for Mac: 1.12.0-beta21 (Build 11019)
macOS: Version 10.11.5 (Build 15F34)
[OK]     docker-cli
[OK]     app
[OK]     moby-syslog
[OK]     disk
[OK]     virtualization
[OK]     system
[OK]     menubar
[OK]     osxfs
[OK]     db
[OK]     slirp
[OK]     moby-console
[OK]     logs
[OK]     vmnetd
[OK]     env
[OK]     moby
[OK]     driver.amd64-linux
```


pmset output:

```
Active Profiles:
Battery Power		-1
AC Power		-1*
Currently in use:
 standbydelay         10800
 standby              1
 womp                 1
 halfdim              1
 hibernatefile        /var/vm/sleepimage
 powernap             1
 gpuswitch            2
 networkoversleep     0
 disksleep            0
 sleep                0 (sleep prevented by coreaudiod)
 autopoweroffdelay    14400
 hibernatemode        3
 autopoweroff         1
 ttyskeepawake        1
 displaysleep         10
 acwake               0
 lidwake              1
```

**Comment by pgraci** [comment]
Same situation as @sambernard this is happening multiple times a day while I am using Docker after latest update.  

Diagnostic ID: 19447443-5A44-4FB7-81E7-D19A4835A7E4
Docker for Mac: 1.12.0-beta21 (Build 11019)
macOS: Version 10.11.3 (Build 15D21)
[OK]     docker-cli
[OK]     app
[OK]     moby-syslog
[OK]     disk
[OK]     virtualization
[OK]     system
[OK]     menubar
[OK]     osxfs
[OK]     db
[OK]     slirp
[OK]     moby-console
[OK]     logs
[OK]     vmnetd
[OK]     env
[OK]     moby
[OK]     driver.amd64-linux

Active Profiles:
Battery Power		-1
AC Power		2*
Currently in use:
 standbydelay         10800
 standby              1
 womp                 1
 halfdim              1
 hibernatefile        /var/vm/sleepimage
 gpuswitch            2
 powernap             1
 networkoversleep     0
 disksleep            10
 sleep                1 (sleep prevented by coreaudiod, Google Chrome)
 autopoweroffdelay    14400
 hibernatemode        3
 autopoweroff         1
 ttyskeepawake        1
 displaysleep         10
 acwake               0
 lidwake              1

**Comment by v4tech** [comment]
I don't know if this helps, but there seems to be a minimum amount of time the machine needs to be suspended...  A short suspend-wake doesn't seem to always trigger this bug.

Hi @amilstead, thanks, for your comments:

Re: ``sudo pmset -g``
```
Active Profiles:
Battery Power           1
AC Power                -1*
Currently in use:
 standbydelay         10800
 standby              1
 womp                 1
 halfdim              1
 hibernatefile        /var/vm/sleepimage
 darkwakes            0
 gpuswitch            2
 networkoversleep     0
 disksleep            10
 sleep                10
 autopoweroffdelay    14400
 hibernatemode        3
 autopoweroff         1
 ttyskeepawake        1
 displaysleep         10
 acwake               0
 lidwake              1
```

Re: my '**don't use it**' - there's two things to be taken from that:
1. If you **need** to have docker running (for work say) then a work around solution is required...
2. I think it's actually quite helpful, knowing that one virtualisation solution (virtualbox in my case) works but the native one doesn't.  It should be somewhat useful for debugging...

**Comment by gumho** [comment]
I'm seeing this issue on my Mac with the latest Docker for Mac as well. Happens when I close my laptop lid for even a few minutes.

**Comment by jeff-xoom** [comment]
I see the same issue only after a few seconds of sleep.

```
Docker for Mac: version: mac-v1.12.0.1
OS X: version 10.10.5 (build: 14F1909)
logs: /tmp/264231A3-7008-4FAA-9D89-83D089716817/20160817-135639.tar.gz
[OK]     docker-cli
[OK]     app
[OK]     moby-syslog
[OK]     disk
[OK]     virtualization
[OK]     system
[OK]     menubar
[OK]     osxfs
[OK]     db
[OK]     slirp
[OK]     moby-console
[OK]     logs
[OK]     vmnetd
[OK]     env
[OK]     moby
[OK]     driver.amd64-linux
```

pmset

```
System-wide power settings:
 DestroyFVKeyOnStandby		0
Active Profiles:
Battery Power		-1
AC Power		2*
Currently in use:
 standbydelay         4200
 standby              1
 womp                 1
 halfdim              1
 hibernatefile        /var/vm/sleepimage
 gpuswitch            2
 darkwakes            1
 networkoversleep     0
 disksleep            10
 sleep                10
 autopoweroffdelay    14400
 hibernatemode        3
 autopoweroff         1
 ttyskeepawake        1
 displaysleep         10
 acwake               0
 lidwake              1
```

**Comment by jorge-3** [comment]
I am seeing this all the time after the update, no suspend necessary. I've closed and restarted Docker client for Mac without any luck:

```
Exception in thread Thread-4:
Traceback (most recent call last):
  File "threading.py", line 810, in __bootstrap_inner
  File "threading.py", line 763, in run
  File "compose/cli/log_printer.py", line 149, in tail_container_logs
  File "compose/cli/log_printer.py", line 179, in wait_on_exit
  File "compose/container.py", line 239, in wait
  File "site-packages/docker/utils/decorators.py", line 21, in wrapped
  File "site-packages/docker/api/container.py", line 441, in wait
  File "site-packages/docker/client.py", line 174, in _raise_for_status
APIError: 500 Server Error: Internal Server Error ("dial unix /Users/jorge/Library/Containers/com.docker.docker/Data/*00000003.00000948: connect: connection refused")

Exception in thread Thread-3:
Traceback (most recent call last):
  File "threading.py", line 810, in __bootstrap_inner
  File "threading.py", line 763, in run
  File "compose/cli/log_printer.py", line 149, in tail_container_logs
  File "compose/cli/log_printer.py", line 179, in wait_on_exit
  File "compose/container.py", line 239, in wait
  File "site-packages/docker/utils/decorators.py", line 21, in wrapped
  File "site-packages/docker/api/container.py", line 441, in wait
  File "site-packages/docker/client.py", line 174, in _raise_for_status
APIError: 500 Server Error: Internal Server Error ("dial unix /Users/jorge/Library/Containers/com.docker.docker/Data/*00000003.00000948: connect: connection refused")
```

**Comment by jamesdixon** [comment]
Is there any update on this issue? This is a real killer.

**Comment by dsheets** [comment]
We have not been able to reproduce this issue, unfortunately. Any and all information about your specific environment would be helpful. We are working on adding more diagnostic information regarding power events that may help us track down the cause in a future release.

**Comment by jamesdixon** [comment]
> We have not been able to reproduce this issue

That is a true bummer 😢 

Here's my diagnostic output:

```
Docker for Mac: version: 1.12.1-beta24 (0afa5e5)
OS X: version 10.10.5 (build: 14F1909)
logs: /tmp/9ED5A2D8-F249-4F5C-84F3-EB0EB3640807/20160901-124227.tar.gz
[OK]     docker-cli
[OK]     virtualization kern.hv_support
[OK]     menubar
[OK]     moby-syslog
[OK]     dns
[OK]     disk
[OK]     system
[OK]     app
[OK]     osxfs
[OK]     virtualization VT-X
[OK]     db
[OK]     slirp
[OK]     logs
[OK]     env
[OK]     vmnetd
[OK]     moby-console
[OK]     moby
[OK]     driver.amd64-linux
```

And power settings:

```
Active Profiles:
Battery Power  		-1
AC Power       		-1*
Currently in use:
 standbydelay         10800
 standby              1
 womp                 1
 halfdim              1
 hibernatefile        /var/vm/sleepimage
 darkwakes            1
 networkoversleep     0
 disksleep            10
 sleep                1
 autopoweroffdelay    14400
 hibernatemode        3
 autopoweroff         1
 ttyskeepawake        1
 displaysleep         10
 acwake               0
 lidwake              1
```

**Comment by jamesdixon** [comment]
Also, I tried setting `hibernatemode` to `0` with no luck; the issue still persisted.

In terms of time before I see the issue occur, it's hard to say, but I can say that I've seen it happen in as little as 15 - 20 minutes after my machine sleeps.

**Comment by v4tech** [comment]
@dsheets - What info would you like?

I agree with @jamesdixon that this is a bummer, but as there are just **soooo** many of us with this problem we surely can get enough info for you.

Going out on a limb with additional info:

I don't know if this will be of assistance, but I've got a whole lot of homebrew packages installed - including `coreutils` replacing the mac defaults.

Does anyone/everyone else use homebrew?  More importantly, does anyone have a base install to test on?

**Comment by gumho** [comment]
@v4tech I and all my colleagues who are experiencing the same problem are also using HomeBrew too

**Comment by jamesdixon** [comment]
I also have homebrew installed, but not heavily used whatsoever.

Looking at everyone's responses, I was curious if this has something to do specifically with the Hypervisor implementation in Yosemite, but it also appears there were a couple who were using El Capitain.

**Comment by djs55** [comment]
In diagnostic bundle `264231A3-7008-4FAA-9D89-83D089716817` It looks like the vCPU freezing wakeup code is triggering hyperkit [vmexit_vmx](https://github.com/docker/hyperkit/blob/547caeb5facb248067c529dd8c80931dbc1c56c6/src/xhyve.c#L424) to `VMEXIT_ABORT`: (hostname and username redacted)
```
Aug 17 14:59:30 Docker[com.docker.driver.amd64-linux][27183] <Notice>: System wants to go to sleep
Aug 17 14:59:30 Docker[com.docker.driver.amd64-linux][27183] <Notice>: Asking com.docker.hyperkit to freeze vcpus
Aug 17 14:59:31 Docker[com.docker.driver.amd64-linux][27183] <Notice>: vcpu 3 waiting for signal to resume
Aug 17 14:59:31 Docker[com.docker.driver.amd64-linux][27183] <Notice>: vcpu 0 waiting for signal to resume
Aug 17 14:59:31 Docker[com.docker.driver.amd64-linux][27183] <Notice>: vcpu freeze complete: allowing sleep
Aug 17 14:59:31 kernel[kern][0] <Notice>: PM response took 105 ms (27183, com.docker.drive)
Aug 17 14:59:31 Docker[com.docker.driver.amd64-linux][27183] <Notice>: vcpu 1 waiting for signal to resume
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>: System wants to wake up
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>: Asking com.docker.hyperkit to thaw vcpus
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>: vcpu 3 received signal, resuming
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>: vm exit[3]
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>:       reason          VMX
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>:       rip             0xffffffff8103b2e4
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>:       inst_length     3
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>:       status          0
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>: virtio-net-vpnkit: initialising, opts="uuid=...
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>:       exit_reason     2
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>: Interface will have uuid 7916572e-0a15-49fc-93a3-b8bb31c3ed15
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>:       qualification   0x0000000000000000
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>:       inst_type               0
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>:       inst_error              0
Aug 17 15:00:10 Docker[com.docker.driver.amd64-linux][27183] <Notice>: Connection established with MAC=c0:ff:ee:c0:ff:ee and MTU 1500
```

**Comment by djs55** [comment]
It looks like at least some cases can be explained by a bug in the workaround for a hypervisor.framework ACPI crash bug. We're not sure whether the workaround is still needed -- the underlying crash may well have been fixed by Apple, especially in the most recent versions and updates of MacOS. If you want to experiment, you can disable the buggy workaround by:

```
cd ~/Library/Containers/com.docker.docker/Data/database
git reset --hard
echo "please do not freeze" > com.docker.driver.amd64-linux/on-sleep
git commit -a -m "disable freeze workaround"
```
After this, when you suspend your machine you should see logs like this (for reference I tried this on beta 25)
```
Sep  9 11:19:00 Docker[68709] <Notice>: System wants to go to sleep
Sep  9 11:19:00 Docker[68709] <Notice>: com.docker.hyperkit is running and not shutting down because on-sleep = please do not freeze
Sep  9 11:19:10 Docker[68709] <Notice>: System wants to wake up
Sep  9 11:19:10 Docker[68709] <Notice>: com.docker.hyperkit is already running so nothing to do
```

If you do try this, please report back what happens, including the diagnostic ID.

Thanks everyone for your patience.

**Comment by elyulka** [comment]
I also had this issue long time on 10.10 and have it after upgrade on 10.12. 
Diagnostic ID: 0E5A3B2C-C92E-471D-8D8A-09328930158A
```
Docker for Mac: version: 1.12.3-beta30 (7314181)
OS X: version 10.12.1 (build: 16B2657)
logs: /tmp/0E5A3B2C-C92E-471D-8D8A-09328930158A/20161111-143255.tar.gz
[OK]     vmnetd
[OK]     dns
[OK]     driver.amd64-linux
[OK]     virtualization VT-X
[OK]     app
[OK]     moby
[OK]     system
[OK]     moby-syslog
[OK]     db
[OK]     env
[OK]     virtualization kern.hv_support
[OK]     slirp
[OK]     osxfs
[OK]     moby-console
[OK]     logs
[OK]     docker-cli
[OK]     menubar
[OK]     disk
```

One of crash reports:
```
Process:               com.docker.hyperkit [4204]
Path:                  /Applications/Docker.app/Contents/MacOS/com.docker.hyperkit
Identifier:            com.docker.hyperkit
Version:               0
Code Type:             X86-64 (Native)
Parent Process:        com.docker.driver.amd64-linux [4200]
Responsible:           Docker [1397]
User ID:               502

Date/Time:             2016-11-09 22:10:12.526 +0200
OS Version:            Mac OS X 10.10.5 (14F2009)
Report Version:        11
Anonymous UUID:        FA3F6567-10D1-A382-EC6D-9D0C4E2EC75E

Sleep/Wake UUID:       5A6CCE1C-CD6A-4FD1-ACEF-EBE495831B79

Time Awake Since Boot: 14000 seconds
Time Since Wake:       1 seconds

Crashed Thread:        12  vcpu:1

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000

Application Specific Information:
abort() called

Thread 0:: Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	0x00007fff96eae21a kevent + 10
1   libdyld.dylib                 	0x00007fff975ad5c9 start + 1

Thread 1:
0   libsystem_kernel.dylib        	0x00007fff96ead3fa __select + 10
1   com.docker.hyperkit           	0x00000001058aefa3 camlLwt_engine__fun_2749 + 35
2   com.docker.hyperkit           	0x00000001058aeb9a camlLwt_engine__fun_2688 + 442
3   com.docker.hyperkit           	0x00000001058b06e8 camlLwt_main__run_1324 + 136
4   com.docker.hyperkit           	0x00000001058f3a89 camlThread__fun_1562 + 137
5   com.docker.hyperkit           	0x000000010596c5e8 caml_start_program + 92
6   com.docker.hyperkit           	0x000000010594eddb caml_thread_start + 107
7   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
8   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
9   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 2:
0   libsystem_kernel.dylib        	0x00007fff96ead3fa __select + 10
1   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
2   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
3   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 3:
0   libsystem_kernel.dylib        	0x00007fff96ead136 __psynch_cvwait + 10
1   com.docker.hyperkit           	0x000000010594bc5b worker_loop + 123
2   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
3   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
4   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 4:: callout
0   libsystem_kernel.dylib        	0x00007fff96ead136 __psynch_cvwait + 10
1   com.docker.hyperkit           	0x0000000105979645 callout_thread_func + 195
2   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
3   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
4   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 5:: net:ipc:tx
0   libsystem_kernel.dylib        	0x00007fff96ead136 __psynch_cvwait + 10
1   com.docker.hyperkit           	0x00000001059925dd pci_vtnet_tx_thread.859 + 277
2   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
3   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
4   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 6:: net:ipc:rx
0   libsystem_kernel.dylib        	0x00007fff96ead3fa __select + 10
1   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
2   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
3   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 7:: blk:3:0
0   libsystem_kernel.dylib        	0x00007fff96ead136 __psynch_cvwait + 10
1   com.docker.hyperkit           	0x0000000105985589 blockif_thr + 256
2   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
3   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
4   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 8:: vsock:tx
0   libsystem_kernel.dylib        	0x00007fff96ead3fa __select + 10
1   com.docker.hyperkit           	0x0000000105993fa4 pci_vtsock_tx_thread + 903
2   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
3   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
4   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 9:: vsock:rx
0   libsystem_kernel.dylib        	0x00007fff96ead3fa __select + 10
1   com.docker.hyperkit           	0x0000000105994ff2 pci_vtsock_rx_thread + 822
2   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
3   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
4   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 10:: Dispatch queue: com.apple.libdispatch-manager
0   libsystem_kernel.dylib        	0x00007fff96eae232 kevent64 + 10
1   libdispatch.dylib             	0x00007fff8dba626e _dispatch_mgr_thread + 52

Thread 11:: vcpu:0
0   libsystem_kernel.dylib        	0x00007fff96ead166 __psynch_mutexwait + 10
1   libsystem_pthread.dylib       	0x00007fff9a63f5ee _pthread_cond_wait + 835
2   com.docker.hyperkit           	0x000000010597d725 vmx_run + 7294
3   com.docker.hyperkit           	0x000000010597aed6 xh_vm_run + 4707
4   com.docker.hyperkit           	0x000000010599a422 vcpu_thread + 1215
5   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
6   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
7   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 12 Crashed:: vcpu:1
0   libsystem_kernel.dylib        	0x00007fff96ead286 __pthread_kill + 10
1   libsystem_c.dylib             	0x00007fff8db6b9a3 abort + 129
2   com.docker.hyperkit           	0x000000010599a494 vcpu_thread + 1329
3   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
4   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
5   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 13:: 9p:db
0   libsystem_kernel.dylib        	0x00007fff96eae682 read + 10
1   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
2   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
3   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 14:: 9p:port
0   libsystem_kernel.dylib        	0x00007fff96eae682 read + 10
1   libsystem_pthread.dylib       	0x00007fff9a63f05a _pthread_body + 131
2   libsystem_pthread.dylib       	0x00007fff9a63efd7 _pthread_start + 176
3   libsystem_pthread.dylib       	0x00007fff9a63c3ed thread_start + 13

Thread 15:
0   libsystem_kernel.dylib        	0x00007fff96ead94a __workq_kernreturn + 10
1   libsystem_pthread.dylib       	0x00007fff9a63c3dd start_wqthread + 13

Thread 12 crashed with X86 Thread State (64-bit):
  rax: 0x0000000000000000  rbx: 0x0000000000000006  rcx: 0x000000018efece38  rdx: 0x0000000000000000
  rdi: 0x0000000000001a03  rsi: 0x0000000000000006  rbp: 0x000000018efece60  rsp: 0x000000018efece38
   r8: 0x000000018efecd20   r9: 0x000000018efed000  r10: 0x0000000008000000  r11: 0x0000000000000206
  r12: 0x0000000105e6f190  r13: 0x0000000105e6f190  r14: 0x000000018efed000  r15: 0x0000000105c42140
  rip: 0x00007fff96ead286  rfl: 0x0000000000000206  cr2: 0x00007fff7c6a9fd8
  
Logical CPU:     0
Error Code:      0x02000148
Trap Number:     133


Binary Images:
       0x1056dc000 -        0x105a4aff7 +com.docker.hyperkit (0) <8F9FE29E-F2E0-369E-BE53-A39D52B49AFA> /Applications/Docker.app/Contents/MacOS/com.docker.hyperkit
       0x1060c1000 -        0x1060c5fff  com.apple.Hypervisor (1.0 - 1) <7960B1D3-1EC1-379D-8CF4-F478BA67CD31> /System/Library/Frameworks/Hypervisor.framework/Versions/A/Hypervisor
       0x1060d0000 -        0x1060d3ffb  com.apple.framework.vmnet (1.0 - 1) <F25FDA16-A497-3D3B-B0DE-2DC8A2B8EE0F> /System/Library/Frameworks/vmnet.framework/Versions/A/vmnet
       0x1060d9000 -        0x1060deff7  com.apple.framework.netrb (1.0 - 1) <2125AEA9-5C9F-34C9-AD6F-5A0248A63227> /System/Library/PrivateFrameworks/Netrb.framework/Versions/A/Netrb
    0x7fff651c7000 -     0x7fff651fd92f  dyld (353.2.4) <708E385E-50C6-32A3-983C-42E97BEE4346> /usr/lib/dyld
    0x7fff8c2b7000 -     0x7fff8c2b9fff  libquarantine.dylib (76.20.1) <7AF90041-2768-378A-925A-D83161863642> /usr/lib/system/libquarantine.dylib
    0x7fff8c397000 -     0x7fff8c60dfff  com.apple.security (7.0 - 57031.40.8) <FB238FFF-AA3B-3AB3-80F8-2DA8D2ADA15A> /System/Library/Frameworks/Security.framework/Versions/A/Security
    0x7fff8c651000 -     0x7fff8c76affb  com.apple.CoreText (454.11 - 454.11) <7E5228C8-8657-3B6C-B3A4-98B79A3041EA> /System/Library/Frameworks/CoreText.framework/Versions/A/CoreText
    0x7fff8c770000 -     0x7fff8c780ff7  libbsm.0.dylib (34) <A3A2E56C-2B65-37C7-B43A-A1F926E1A0BB> /usr/lib/libbsm.0.dylib
    0x7fff8cc97000 -     0x7fff8ccfeffb  com.apple.datadetectorscore (6.0 - 396.1.2) <E5A147AE-66B5-3D31-A2E0-D3FAB8417E1F> /System/Library/PrivateFrameworks/DataDetectorsCore.framework/Versions/A/DataDetectorsCore
    0x7fff8cd47000 -     0x7fff8cd48fff  libDiagnosticMessagesClient.dylib (100) <2EE8E436-5CDC-34C5-9959-5BA218D507FB> /usr/lib/libDiagnosticMessagesClient.dylib
    0x7fff8cd4d000 -     0x7fff8d0e5ff7  com.apple.CoreFoundation (6.9 - 1153.18) <5C0892B8-9691-341F-9279-CA3A74D59AA0> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
    0x7fff8d0e6000 -     0x7fff8d2f6ff7  com.apple.CFNetwork (720.5.7 - 720.5.7) <A47D7DD2-5903-39D0-8D27-5A7E2D119D84> /System/Library/Frameworks/CFNetwork.framework/Versions/A/CFNetwork
    0x7fff8d2f7000 -     0x7fff8d4f146f  libobjc.A.dylib (647) <759E155D-BC42-3D4E-869B-6F57D477177C> /usr/lib/libobjc.A.dylib
    0x7fff8d996000 -     0x7fff8d99dff7  libcompiler_rt.dylib (35) <BF8FC133-EE10-3DA6-9B90-92039E28678F> /usr/lib/system/libcompiler_rt.dylib
    0x7fff8db0e000 -     0x7fff8db9afef  libsystem_c.dylib (1044.40.3) <2E673D96-E334-37D2-833A-E57383BBA377> /usr/lib/system/libsystem_c.dylib
    0x7fff8db9b000 -     0x7fff8db9ffff  com.apple.TCC (1.0 - 1) <CCA42EE2-3400-3444-9486-BC454E60D944> /System/Library/PrivateFrameworks/TCC.framework/Versions/A/TCC
    0x7fff8dba0000 -     0x7fff8dbcaff7  libdispatch.dylib (442.50.1) <A61E703C-784A-3698-B513-75DD12AAD6DC> /usr/lib/system/libdispatch.dylib
    0x7fff8dbcb000 -     0x7fff8dbcdff7  libsystem_coreservices.dylib (9) <41B7C578-5A53-31C8-A96F-C73E030B0938> /usr/lib/system/libsystem_coreservices.dylib
    0x7fff8dc2e000 -     0x7fff8dc3ffff  libcmph.dylib (1) <46EC3997-DB5E-38AE-BBBB-A035A54AD3C0> /usr/lib/libcmph.dylib
    0x7fff8dc7a000 -     0x7fff8dc7cfff  libRadiance.dylib (1247) <0028A2E3-7928-3479-9BF7-CB64234F85AF> /System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libRadiance.dylib
    0x7fff8dce4000 -     0x7fff8dce6fff  libsystem_configuration.dylib (699.40.2) <56F94DCE-DBDE-3615-8F07-DE6270D9F8BE> /usr/lib/system/libsystem_configuration.dylib
    0x7fff8dd1f000 -     0x7fff8ddbee27  com.apple.AppleJPEG (1.0 - 1) <6627DDD9-A8FE-3968-B23A-B6A29AA3919A> /System/Library/PrivateFrameworks/AppleJPEG.framework/Versions/A/AppleJPEG
    0x7fff8de46000 -     0x7fff8de92ff7  libcups.2.dylib (408.2) <E8AD18F9-61E4-3791-B840-504468C25556> /usr/lib/libcups.2.dylib
    0x7fff8de93000 -     0x7fff8de95fff  com.apple.loginsupport (1.0 - 1) <DAAD7013-A19D-3858-BFF7-DE1DAF664401> /System/Library/PrivateFrameworks/login.framework/Versions/A/Frameworks/loginsupport.framework/Versions/A/loginsupport
    0x7fff8de96000 -     0x7fff8dfdcfef  libsqlite3.dylib (168.2) <53F6A294-15D7-3804-9ABF-47D35E15CDFB> /usr/lib/libsqlite3.dylib
    0x7fff8e00b000 -     0x7fff8e04bff7  libGLImage.dylib (11.1.6) <D6DDC014-2C74-34B3-970A-26F211FF76EF> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGLImage.dylib
    0x7fff8e10e000 -     0x7fff8e154ff7  libFontRegistry.dylib (134.1) <CE41D8C2-BEED-345C-BC4F-3775CC06C672> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/ATS.framework/Versions/A/Resources/libFontRegistry.dylib
    0x7fff8e214000 -     0x7fff8e308ff7  libFontParser.dylib (134.10) <6CF434A2-91CA-3477-BDE2-124508EDA05D> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/ATS.framework/Versions/A/Resources/libFontParser.dylib
    0x7fff8e337000 -     0x7fff8e338ff7  libsystem_blocks.dylib (65) <9615D10A-FCA7-3BE4-AA1A-1B195DACE1A1> /usr/lib/system/libsystem_blocks.dylib
    0x7fff8e4a2000 -     0x7fff8e4adfff  libcommonCrypto.dylib (60061.30.1) <E789748D-F9A7-3CFF-B317-90DF348B1E95> /usr/lib/system/libcommonCrypto.dylib
    0x7fff8e4b2000 -     0x7fff8e52aff7  com.apple.SystemConfiguration (1.14.4 - 1.14) <3DFFD7F7-BD23-3F4C-A209-C4A0D99F6573> /System/Library/Frameworks/SystemConfiguration.framework/Versions/A/SystemConfiguration
    0x7fff8f8ca000 -     0x7fff8f8fafff  libsystem_m.dylib (3086.1) <1E12AB45-6D96-36D0-A226-F24D9FB0D9D6> /usr/lib/system/libsystem_m.dylib
    0x7fff8fa4e000 -     0x7fff8fabdfff  com.apple.SearchKit (1.4.0 - 1.4.0) <80883BD1-C9BA-3794-A20E-476F94DD89A9> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/SearchKit.framework/Versions/A/SearchKit
    0x7fff8fabe000 -     0x7fff8fac7ff3  com.apple.CommonAuth (4.0 - 2.0) <9A484EE6-0003-3AB1-AE4F-AA543BBBF53F> /System/Library/PrivateFrameworks/CommonAuth.framework/Versions/A/CommonAuth
    0x7fff8fbc4000 -     0x7fff8fbd5ff7  libz.1.dylib (55) <88C7C7DE-04B8-316F-8B74-ACD9F3DE1AA1> /usr/lib/libz.1.dylib
    0x7fff8fcb3000 -     0x7fff8fcbffff  com.apple.speech.synthesis.framework (5.3.11 - 5.3.11) <DFC7FD85-F1B0-317C-8513-722570CB8FB9> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/SpeechSynthesis.framework/Versions/A/SpeechSynthesis
    0x7fff8fcc0000 -     0x7fff8fcc0fff  com.apple.Accelerate.vecLib (3.10 - vecLib 3.10) <9D749502-A228-3BF1-B52F-A182DEEB2C4D> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/vecLib
    0x7fff8fcc1000 -     0x7fff8fdfcfff  com.apple.ImageIO.framework (3.3.0 - 1247) <F708DED7-DF04-3D82-A88C-FD85D7D29CFF> /System/Library/Frameworks/ImageIO.framework/Versions/A/ImageIO
    0x7fff8fdfd000 -     0x7fff8fe22fff  libPng.dylib (1247) <1E8B06AA-1DC5-3845-88D9-B2F8E6E1E540> /System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libPng.dylib
    0x7fff8fe23000 -     0x7fff8fec1fff  com.apple.Metadata (10.7.0 - 917.36) <00C4CB5D-E723-3612-84E0-439098392CDD> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/Metadata.framework/Versions/A/Metadata
    0x7fff8fec2000 -     0x7fff8ffb4ff7  libxml2.2.dylib (26.12) <769FD755-D24A-3A7C-ABEE-9A82BB1196D0> /usr/lib/libxml2.2.dylib
    0x7fff9010e000 -     0x7fff90137ff3  libxslt.1.dylib (13.12) <19B2D915-DC29-3F79-9756-D7EB9051FEFF> /usr/lib/libxslt.1.dylib
    0x7fff9027d000 -     0x7fff90281fff  libCoreVMClient.dylib (79.1) <201EF6DF-5074-3CB7-A361-398CF957A264> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libCoreVMClient.dylib
    0x7fff908bb000 -     0x7fff908d2ff7  libLinearAlgebra.dylib (1128) <E78CCBAA-A999-3B65-8EC9-06DB15E67C37> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libLinearAlgebra.dylib
    0x7fff908d3000 -     0x7fff908d4fff  liblangid.dylib (117) <B54A4AA0-2E53-3671-90F5-AFF711C0EB9E> /usr/lib/liblangid.dylib
    0x7fff908dc000 -     0x7fff90b5bff7  com.apple.CoreData (111 - 526.3) <5A27E0D8-5E5A-335B-B3F6-2601C7B976FA> /System/Library/Frameworks/CoreData.framework/Versions/A/CoreData
    0x7fff90bac000 -     0x7fff90bf6fff  com.apple.HIServices (1.22 - 523) <99ADA325-53C8-368E-A5FA-3F913CD2245A> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/HIServices.framework/Versions/A/HIServices
    0x7fff90bf7000 -     0x7fff90c75fff  com.apple.CoreServices.OSServices (640.4 - 640.4) <9D175CD9-B27F-3529-8269-66D8AA83582E> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/OSServices.framework/Versions/A/OSServices
    0x7fff90c82000 -     0x7fff90c8afff  libsystem_dnssd.dylib (576.50.2) <9EC5AF92-D0D2-3BDE-92B6-D3730D3865C8> /usr/lib/system/libsystem_dnssd.dylib
    0x7fff912a3000 -     0x7fff912a6fff  com.apple.xpc.ServiceManagement (1.0 - 1) <A95A15CD-3B21-3513-AFF8-1D7DE3DBFA12> /System/Library/Frameworks/ServiceManagement.framework/Versions/A/ServiceManagement
    0x7fff912a7000 -     0x7fff9158effb  com.apple.CoreServices.CarbonCore (1108.6 - 1108.6) <8953580E-7857-33B2-AA64-98296830D3A8> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/CarbonCore.framework/Versions/A/CarbonCore
    0x7fff9158f000 -     0x7fff915bafff  libc++abi.dylib (125) <88A22A0F-87C6-3002-BFBA-AC0F2808B8B9> /usr/lib/libc++abi.dylib
    0x7fff915bc000 -     0x7fff915d2ff7  libsystem_asl.dylib (267) <F153AC5B-0542-356E-88C8-20A62CA704E2> /usr/lib/system/libsystem_asl.dylib
    0x7fff92960000 -     0x7fff931a4ffb  com.apple.CoreGraphics (1.600.0 - 788.3.5) <83491927-C5F2-3E1D-9C01-84BC802AEFF3> /System/Library/Frameworks/CoreGraphics.framework/Versions/A/CoreGraphics
    0x7fff936b5000 -     0x7fff936bbfff  libsystem_trace.dylib (72.20.1) <840F5301-B55A-3078-90B9-FEFFD6CD741A> /usr/lib/system/libsystem_trace.dylib
    0x7fff936bc000 -     0x7fff936c8ff7  com.apple.OpenDirectory (10.10 - 187) <22FF4721-03D8-381B-AA83-2B07D690C777> /System/Library/Frameworks/OpenDirectory.framework/Versions/A/OpenDirectory
    0x7fff936c9000 -     0x7fff937bafff  libJP2.dylib (1247) <D74A8708-C27E-3D7C-89D0-BE999596E2B1> /System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libJP2.dylib
    0x7fff937bb000 -     0x7fff937d5ff7  com.apple.Kerberos (3.0 - 1) <7760E0C2-A222-3709-B2A6-B692D900CEB1> /System/Library/Frameworks/Kerberos.framework/Versions/A/Kerberos
    0x7fff9386d000 -     0x7fff9386dff7  liblaunch.dylib (559.40.1) <4F81CA3A-D2CE-3030-A89D-42F3DAD7BA8F> /usr/lib/system/liblaunch.dylib
    0x7fff939e3000 -     0x7fff939e8fff  libsystem_stats.dylib (163.30.2) <CDB327EF-3C80-362F-9BF6-718CBBC5DB24> /usr/lib/system/libsystem_stats.dylib
    0x7fff939e9000 -     0x7fff93a3afff  com.apple.audio.CoreAudio (4.3.0 - 4.3.0) <8E6AE326-9F3E-3182-BD9E-FDF561211E2E> /System/Library/Frameworks/CoreAudio.framework/Versions/A/CoreAudio
    0x7fff93a52000 -     0x7fff93a53ffb  libremovefile.dylib (35) <3485B5F4-6CE8-3C62-8DFD-8736ED6E8531> /usr/lib/system/libremovefile.dylib
    0x7fff93a87000 -     0x7fff93babff7  com.apple.LaunchServices (644.56 - 644.56) <20AABB1C-9319-3E4D-A024-51B0DD5FCD3B> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/LaunchServices.framework/Versions/A/LaunchServices
    0x7fff93bac000 -     0x7fff93c00fff  libc++.1.dylib (120) <1B9530FD-989B-3174-BB1C-BDC159501710> /usr/lib/libc++.1.dylib
    0x7fff93dd8000 -     0x7fff93ddeff7  libsystem_networkextension.dylib (167.40.3) <BA58B30B-8377-3B0A-8AE3-4F84021D9D4E> /usr/lib/system/libsystem_networkextension.dylib
    0x7fff93e9f000 -     0x7fff93ea6fff  com.apple.NetFS (6.0 - 4.0) <C263C8F8-F284-3101-AC82-A97A81716063> /System/Library/Frameworks/NetFS.framework/Versions/A/NetFS
    0x7fff943b6000 -     0x7fff943e1ffb  libarchive.2.dylib (30.50.1) <C140763F-5470-3302-B3E0-651B4B5930ED> /usr/lib/libarchive.2.dylib
    0x7fff94411000 -     0x7fff9442bff7  libextension.dylib (55.2) <3BB019CA-199A-36AC-AA22-14B562138545> /usr/lib/libextension.dylib
    0x7fff94453000 -     0x7fff94460ff7  libbz2.1.0.dylib (36) <2DF83FBC-5C08-39E1-94F5-C28653791B5F> /usr/lib/libbz2.1.0.dylib
    0x7fff94477000 -     0x7fff9447fffb  libcopyfile.dylib (118.1.2) <0C68D3A6-ACDD-3EF3-991A-CC82C32AB836> /usr/lib/system/libcopyfile.dylib
    0x7fff9453d000 -     0x7fff945affff  com.apple.framework.IOKit (2.0.2 - 1050.20.2) <09C0518C-90DF-3FC3-96D6-34D35F72C8EF> /System/Library/Frameworks/IOKit.framework/Versions/A/IOKit
    0x7fff945b0000 -     0x7fff945bbff7  libkxld.dylib (2782.50.9) <D0957933-18F0-3058-B5E4-F33AB3D6FAEC> /usr/lib/system/libkxld.dylib
    0x7fff94866000 -     0x7fff948a1fff  com.apple.QD (301 - 301) <C4D2AD03-B839-350A-AAF0-B4A08F8BED77> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/QD.framework/Versions/A/QD
    0x7fff957a5000 -     0x7fff957c0ff7  libCRFSuite.dylib (34) <D64842BE-7BD4-3D0C-9842-1D202F7C2A51> /usr/lib/libCRFSuite.dylib
    0x7fff957c1000 -     0x7fff958b3ff7  libiconv.2.dylib (42) <2A06D02F-8B76-3864-8D96-64EF5B40BC6C> /usr/lib/libiconv.2.dylib
    0x7fff958cc000 -     0x7fff958e9ffb  libresolv.9.dylib (57) <26B38E61-298A-3C3A-82C1-3B5E98AD5E29> /usr/lib/libresolv.9.dylib
    0x7fff95f42000 -     0x7fff95f47ffb  libheimdal-asn1.dylib (398.40.1) <7D2BE3DE-60F7-3A6E-A92E-DA0EF9D3417E> /usr/lib/libheimdal-asn1.dylib
    0x7fff963a5000 -     0x7fff963a9ff7  libGIF.dylib (1247) <5EA6D19A-9922-39F6-82DB-DE7A5B8860CF> /System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libGIF.dylib
    0x7fff963aa000 -     0x7fff96459fe7  libvMisc.dylib (516) <6739E390-46E7-3BFA-9B69-B278562326E6> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libvMisc.dylib
    0x7fff96483000 -     0x7fff964fcfe7  libcorecrypto.dylib (233.30.1) <5779FFA0-4D9A-3AD4-B7F2-618227621DC8> /usr/lib/system/libcorecrypto.dylib
    0x7fff9659b000 -     0x7fff965bffef  libJPEG.dylib (1247) <78AA0479-176D-3C69-ADFB-4609A4A676FF> /System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libJPEG.dylib
    0x7fff965dd000 -     0x7fff9664effb  com.apple.ApplicationServices.ATS (360 - 375.4) <888FE0B1-835C-30B1-80AD-5066CADB0364> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/ATS.framework/Versions/A/ATS
    0x7fff9664f000 -     0x7fff96657fff  libsystem_platform.dylib (63) <64E34079-D712-3D66-9CE2-418624A5C040> /usr/lib/system/libsystem_platform.dylib
    0x7fff96733000 -     0x7fff96733fff  libOpenScriptingUtil.dylib (162.2) <D6A2216D-ADB2-3F24-AD30-F6D00829F545> /usr/lib/libOpenScriptingUtil.dylib
    0x7fff967b0000 -     0x7fff967dbff7  com.apple.DictionaryServices (1.2 - 229.1) <62EC3E1B-5A28-3252-90FF-C2E9999C2A2A> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/DictionaryServices.framework/Versions/A/DictionaryServices
    0x7fff967dc000 -     0x7fff967e5ff7  libsystem_notify.dylib (133.1.1) <61147800-F320-3DAA-850C-BADF33855F29> /usr/lib/system/libsystem_notify.dylib
    0x7fff967e6000 -     0x7fff967ffff7  com.apple.CFOpenDirectory (10.10 - 187) <F1805A0F-D5CC-35A8-9349-042D58D128B0> /System/Library/Frameworks/OpenDirectory.framework/Versions/A/Frameworks/CFOpenDirectory.framework/Versions/A/CFOpenDirectory
    0x7fff96d35000 -     0x7fff96d38fff  com.apple.IOSurface (97.4 - 97.4) <AE11CFBC-4D46-30F3-BEEC-4C8131079391> /System/Library/Frameworks/IOSurface.framework/Versions/A/IOSurface
    0x7fff96e97000 -     0x7fff96eb4fff  libsystem_kernel.dylib (2782.50.9) <C09DE9E0-DFDC-345A-8CF7-97CCB62168A6> /usr/lib/system/libsystem_kernel.dylib
    0x7fff96eb5000 -     0x7fff96fc7ff7  libvDSP.dylib (516) <151B3CCB-77D3-3715-A3D0-7C74CD5C7FFC> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libvDSP.dylib
    0x7fff9704b000 -     0x7fff9708cfff  libGLU.dylib (11.1.6) <4BC6579D-A6D0-3A19-880E-E29FAEA698E7> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGLU.dylib
    0x7fff973b9000 -     0x7fff973ffff7  libauto.dylib (186) <A260789B-D4D8-316A-9490-254767B8A5F1> /usr/lib/libauto.dylib
    0x7fff97409000 -     0x7fff97570ffb  com.apple.audio.toolbox.AudioToolbox (1.12 - 1.12) <5678FC94-456A-3F5F-BA9A-10EB6E462997> /System/Library/Frameworks/AudioToolbox.framework/Versions/A/AudioToolbox
    0x7fff975aa000 -     0x7fff975adff7  libdyld.dylib (353.2.4) <224F35C5-6E35-3EC4-A97C-80949CE444E7> /usr/lib/system/libdyld.dylib
    0x7fff975ae000 -     0x7fff975d6fff  libxpc.dylib (559.40.1) <5C829202-962E-3744-8B50-00D38CC88E84> /usr/lib/system/libxpc.dylib
    0x7fff975e5000 -     0x7fff975eefff  libGFXShared.dylib (11.1.6) <CB24B5DA-CB17-3A0B-94E1-5A58DADC3FBD> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGFXShared.dylib
    0x7fff97605000 -     0x7fff9769aff7  com.apple.ColorSync (4.9.0 - 4.9.0) <9150C2B7-2E6E-3509-96EA-7B3F959F049E> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/ColorSync.framework/Versions/A/ColorSync
    0x7fff98044000 -     0x7fff980a3fff  com.apple.AE (681.5 - 681.7) <2BF39455-1CDD-392C-824A-9972C6B1FB57> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/AE.framework/Versions/A/AE
    0x7fff98100000 -     0x7fff98128fff  libsystem_info.dylib (459.40.1) <2E16C4B3-A327-3957-9C41-143911979A1E> /usr/lib/system/libsystem_info.dylib
    0x7fff98129000 -     0x7fff98197ff3  com.apple.Heimdal (4.0 - 2.0) <8D1667CF-D454-3E07-A58E-E15591B5A95E> /System/Library/PrivateFrameworks/Heimdal.framework/Versions/A/Heimdal
    0x7fff98263000 -     0x7fff9827dff7  liblzma.5.dylib (7) <1D03E875-A7C0-3028-814C-3C27F7B7C079> /usr/lib/liblzma.5.dylib
    0x7fff9827e000 -     0x7fff9828bfff  libxar.1.dylib (255) <7CD69BB5-97BA-3858-8A8B-2F33F129E6E7> /usr/lib/libxar.1.dylib
    0x7fff984ba000 -     0x7fff984bafff  com.apple.ApplicationServices (48 - 48) <5BF7910B-C328-3BF8-BA4F-CE52B574CE01> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/ApplicationServices
    0x7fff98561000 -     0x7fff9856bff7  com.apple.NetAuth (5.2 - 5.2) <2BBD749A-8E18-35B8-8E48-A90347C1CCA7> /System/Library/PrivateFrameworks/NetAuth.framework/Versions/A/NetAuth
    0x7fff98728000 -     0x7fff98728ff7  libkeymgr.dylib (28) <77845842-DE70-3CC5-BD01-C3D14227CED5> /usr/lib/system/libkeymgr.dylib
    0x7fff988a0000 -     0x7fff988a0fff  com.apple.CoreServices (62 - 62) <C69DA8A7-B536-34BF-A93F-1C170E2C6D58> /System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices
    0x7fff98d0f000 -     0x7fff98d13fff  libcache.dylib (69) <45E9A2E7-99C4-36B2-BEE3-0C4E11614AD1> /usr/lib/system/libcache.dylib
    0x7fff98e6d000 -     0x7fff98e6dff7  libunc.dylib (29) <5676F7EA-C1DF-329F-B006-D2C3022B7D70> /usr/lib/system/libunc.dylib
    0x7fff98f49000 -     0x7fff98f4aff3  libSystem.B.dylib (1213) <4B24B5BE-45F2-355A-9A35-D438A516D140> /usr/lib/libSystem.B.dylib
    0x7fff98f9b000 -     0x7fff98facff3  libsystem_coretls.dylib (35.40.1) <155DA0A9-2046-332E-BFA3-D7974A51F731> /usr/lib/system/libsystem_coretls.dylib
    0x7fff99a01000 -     0x7fff99a0cfff  libGL.dylib (11.1.6) <38B3FF9A-5660-3080-9DB8-1CAE0E8CD739> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGL.dylib
    0x7fff99a32000 -     0x7fff99a34fff  libsystem_sandbox.dylib (358.20.5) <3F5E973F-C702-31AC-97BC-05F5C195683C> /usr/lib/system/libsystem_sandbox.dylib
    0x7fff99a35000 -     0x7fff99a3aff7  libunwind.dylib (35.3) <BE7E51A0-B6EA-3A54-9CCA-9D88F683A6D6> /usr/lib/system/libunwind.dylib
    0x7fff99a3b000 -     0x7fff99a5bff7  com.apple.MultitouchSupport.framework (264.6 - 264.6) <1539F1F6-6334-37F3-9C52-02EFFBF4835D> /System/Library/PrivateFrameworks/MultitouchSupport.framework/Versions/A/MultitouchSupport
    0x7fff99c06000 -     0x7fff99f21fcf  com.apple.vImage (8.0 - 8.0) <1183FE6A-FDB6-3B3B-928D-50C7909F2308> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vImage.framework/Versions/A/vImage
    0x7fff99f22000 -     0x7fff9a107ff7  libicucore.A.dylib (531.50) <CAA7ACE0-4012-3A13-864D-2B56B4B072FD> /usr/lib/libicucore.A.dylib
    0x7fff9a1c2000 -     0x7fff9a21cff7  com.apple.LanguageModeling (1.0 - 1) <ACA93FE0-A0E3-333E-AE3C-8EB7DE5F362F> /System/Library/PrivateFrameworks/LanguageModeling.framework/Versions/A/LanguageModeling
    0x7fff9a21d000 -     0x7fff9a22cfff  com.apple.LangAnalysis (1.7.0 - 1.7.0) <D1E527E4-C561-352F-9457-E8C50232793C> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/LangAnalysis.framework/Versions/A/LangAnalysis
    0x7fff9a22d000 -     0x7fff9a63aff7  libLAPACK.dylib (1128) <F9201AE7-B031-36DB-BCF8-971E994EF7C1> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libLAPACK.dylib
    0x7fff9a63b000 -     0x7fff9a644fff  libsystem_pthread.dylib (105.40.1) <ACE90967-ECD0-3251-AEEB-461E3C6414F7> /usr/lib/system/libsystem_pthread.dylib
    0x7fff9a645000 -     0x7fff9a675ff3  com.apple.GSS (4.0 - 2.0) <97F2A028-44CF-3188-B863-F4EEB39CBDBD> /System/Library/Frameworks/GSS.framework/Versions/A/GSS
    0x7fff9ab3c000 -     0x7fff9ab69fff  com.apple.CoreVideo (1.8 - 145.1) <18DB07E0-B927-3260-A234-636F298D1917> /System/Library/Frameworks/CoreVideo.framework/Versions/A/CoreVideo
    0x7fff9adac000 -     0x7fff9ae07fe7  libTIFF.dylib (1247) <43CB4BDE-84FD-3CCE-96C0-FB33BA2753BD> /System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libTIFF.dylib
    0x7fff9ae60000 -     0x7fff9ae7bfff  com.apple.AppleVPAFramework (1.4.5 - 1.4.5) <B21714FD-3F2E-36D8-996E-EC74676F933F> /System/Library/PrivateFrameworks/AppleVPA.framework/Versions/A/AppleVPA
    0x7fff9ae7c000 -     0x7fff9ae80fff  libpam.2.dylib (20) <E805398D-9A92-31F8-8005-8DC188BD8B6E> /usr/lib/libpam.2.dylib
    0x7fff9aea6000 -     0x7fff9aea6fff  com.apple.Accelerate (1.10 - Accelerate 1.10) <2C8AF258-4F11-3BEC-A826-22D7199B3975> /System/Library/Frameworks/Accelerate.framework/Versions/A/Accelerate
    0x7fff9aea7000 -     0x7fff9aea8fff  libsystem_secinit.dylib (18) <581DAD0F-6B63-3A48-B63B-917AF799ABAA> /usr/lib/system/libsystem_secinit.dylib
    0x7fff9aea9000 -     0x7fff9aeaefff  com.apple.DiskArbitration (2.6 - 2.6) <0DFF4D9B-2AC3-3B82-B5C5-30F4EFBD2DB9> /System/Library/Frameworks/DiskArbitration.framework/Versions/A/DiskArbitration
    0x7fff9af35000 -     0x7fff9af43ff7  com.apple.opengl (11.1.6 - 11.1.6) <DB9F2798-F31F-335E-90A9-992C3AA7ECA8> /System/Library/Frameworks/OpenGL.framework/Versions/A/OpenGL
    0x7fff9af44000 -     0x7fff9afb8ffb  com.apple.securityfoundation (6.0 - 55126) <344401F5-4016-3AEB-BFDC-CE8E5546D39F> /System/Library/Frameworks/SecurityFoundation.framework/Versions/A/SecurityFoundation
    0x7fff9afb9000 -     0x7fff9b147fff  libBLAS.dylib (1128) <497912C1-A98E-3281-BED7-E9C751552F61> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
    0x7fff9b181000 -     0x7fff9b189ff3  com.apple.CoreServices.FSEvents (1210.20.1 - 1210.20.1) <84F79D3E-7B5E-3C93-8479-35794A3F125E> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/FSEvents.framework/Versions/A/FSEvents
    0x7fff9b4a8000 -     0x7fff9b4c4ff7  libsystem_malloc.dylib (53.30.1) <DDA8928B-CC0D-3255-BD8A-3FEA0982B890> /usr/lib/system/libsystem_malloc.dylib
    0x7fff9b4c5000 -     0x7fff9b512ff7  com.apple.print.framework.PrintCore (10.3 - 451.1) <DE992474-0841-38A1-B4F6-46D653E454D5> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/PrintCore.framework/Versions/A/PrintCore
    0x7fff9b528000 -     0x7fff9b560fff  libsystem_network.dylib (412.20.3) <6105C134-6722-3C0A-A4CE-5E1261E2E1CC> /usr/lib/system/libsystem_network.dylib
    0x7fff9b561000 -     0x7fff9b566ff7  libmacho.dylib (862) <126CA2ED-DE91-308F-8881-B9DAEC3C63B6> /usr/lib/system/libmacho.dylib
    0x7fff9b65f000 -     0x7fff9b992ff7  libmecabra.dylib (666.7) <0ED8AE5E-7A5B-34A6-A2EE-2B852E60E1E2> /usr/lib/libmecabra.dylib
    0x7fff9b993000 -     0x7fff9bcc4fff  com.apple.Foundation (6.9 - 1154) <49EE64E1-9F53-35D1-A481-2EFE2789B254> /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation
    0x7fff9bdd6000 -     0x7fff9bdd8fff  libCVMSPluginSupport.dylib (11.1.6) <F82BF8C4-12D3-3EAD-AEDE-533AC08AEB4A> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libCVMSPluginSupport.dylib

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 13
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 87544
    thread_create: 0
    thread_set_state: 0

VM Region Summary:
ReadOnly portion of Libraries: Total=145.1M resident=139.5M(96%) swapped_out_or_unallocated=5700K(4%)
Writable regions: Total=2.2G written=54.8M(2%) resident=447.7M(20%) swapped_out=0K(0%) unallocated=1.8G(80%)
 
REGION TYPE                      VIRTUAL
===========                      =======
Dispatch continuations             8192K
IOKit                               112K
Kernel Alloc Once                     4K
MALLOC                              2.2G
MALLOC (admin)                       32K
STACK GUARD                        56.1M
Stack                              15.2M
VM_ALLOCATE                           8K
VM_ALLOCATE (reserved)               16K        reserved VM address space (unallocated)
__DATA                             10.1M
__LINKEDIT                         73.3M
__TEXT                             71.8M
__UNICODE                           552K
__inout_port_set                      4K
__lpc_dsdt_set                        4K
__lpc_sysres_set                      4K
__pci_devemu_set                      4K
mapped file                          64K
shared memory                         4K
===========                      =======
TOTAL                               2.4G
TOTAL, minus reserved VM space      2.4G
```

**Comment by ijc25** [comment]
@elyulka your issue on 10.12 is not the same as the known 10.10 issue and appears to be a new one (albeit one with a superficially similar symptom). Please would you open a separate ticket so as not to conflate multiple different issues.

**Comment by ijc25** [comment]
@elyulka the issue you quote above was captured on 10.10. I don't see any crashes of 10.12 in your diagnostics ball. In any case please report the 10.12 issue separately.

**Comment by ijc25** [comment]
There seem to be no more reports of the original issue on 10.12 or 10.11.6 since we changed the default and gave the workaround for upgrading users in https://github.com/docker/for-mac/issues/85#issuecomment-245877874.

**Comment by v4tech** [comment]
@ijc25 - I think closing is a bit premature.  There's been no comments from anyone that the fix works.

Your comment uses terms like:
  - _We're not sure whether the workaround is still needed_
  - _If you want to experiment_
They come across as massive red flags, i.e. resulting in a "wait until the (awesome) Docker team tells me this is fixed"...

Are you saying this bug is fixed?

**Comment by ijc25** [comment]
@v4tech yes, we believe this issue is fixed on 10.12 and 10.11.6 now that we have disabled the ACPI crash workaround by default on those versions. If you are upgrading from an older docker for mac then you will need to either follow the steps in https://github.com/docker/for-mac/issues/85#issuecomment-245877874 or perform a factory reset to pickup the new default.

**Comment by c0bra** [comment]
@v4tech @ijc25 I just tried the fix and it worked for me. My containers were not killed while my laptop was closed overnight.

**Comment by c0bra** [comment]
Reversion after latest update :(

**Comment by jscheel** [comment]
I am experiencing this issue while using docker-machine. Is there any connection between the two?

**Comment by dobegor** [comment]
I have this issue running Docker for Mac Edge on OS X 10.12.3.
@ijc25 reopen?

**Comment by chrisamiller** [comment]
I also have this issue with Docker Community Edition v 17.03.1-ce-mac12 (17661) - channel stable.  I've also tried pausing the docker image first, but still get a crash while it's paused:  `(ERRO[0388] error getting events from daemon: EOF)`

**Comment by Ralle** [comment]
Running a Hackintosh on Sierra. My Docker seems to restart after sleep/wake.

Running "Docker 17.06.0-ce-mac18"

**Comment by dobegor** [comment]
^ I have a same effect. @ijc25 please reopen this?

**Comment by darkermatter** [comment]
This is happening to me too:

Version 17.06.0-ce-mac18 (18433)
Channel: stable
d9b66511e0

**Comment by dobegor** [comment]
@ijc this needs to be reopened. I was able to reproduce it on 3 machines. Docker restarts every time machine goes to sleep.

**Comment by djs55** [comment]
@dobegor could you upload diagnostics? Alternatively have a look in your logs for a line like this
```
<Warning>: Triggering a VM reboot because proxy settings changed from 
main.proxySettings{http:"", https:"", exclude:""} to main.proxySettings{http:"", https:"", exclude:"*.local, 169.254/16"}
```
If this line is present then the issue is the same as #1809, a fix for which is scheduled for 17.06.1 hopefully this week. In the meantime a workaround is to disable HTTP proxy tracking by: Whale menu -> Preferences... -> Proxies -> No Proxy.

**Comment by dobegor** [comment]
@djs55 where can I find logs? Diagnostics menu is just hanging for a long time.

**Comment by dobegor** [comment]
@djs55 found it. yes, it was the proxy change. I wonder why the hell it happens when I unplug the charging cable, though. Maybe OS X is doing some things with rearranging type-c connected devices (including type c ethernet bridge I have on the other port).

**Comment by djs55** [comment]
@dobegor thanks for the diagnostics. It does look the same as #1809 to me.

**Comment by daveisfera** [comment]
Mine froze up last night: 91BA0B97-2185-40E4-8CF7-F4C94774E732

**Comment by asoltesz** [comment]
My images still die immediately after suspend/resume.

Docker Version: Version 17.06.0-ce-mac18 (18433)
OSX Version: 10.11.3 El Capitan

It seems that containers even die when you change settings in the Energy Management settings of OSX.

This is a HUGE problem, renders Docker for Mac very-very hard to use.

**Comment by darkermatter** [comment]
I'm now running 17.06.0-ce-mac17 (18432) and after changing the settings to "No proxy" I haven't had any problems.

**Comment by pirminis** [comment]
@darkermatter this solution worked for me too.

**Comment by jedihe** [comment]
The fix posted by @djs55 works for me (tested with manually triggering Sleep and also by closing the lid). 'No proxy' / 'Use system proxy' don't seem to affect it. Longest time I kept the lid closed was ~1 min.

I'm running:

- macOS 10.10.5 (Yosemite)
- Docker version 17.06.0-ce, build 02c1d87

Fix with minor adjustment:

```
cd ~/Library/Containers/com.docker.docker/Data/database
git reset --hard
echo "please do not freeze" > com.docker.driver.amd64-linux/on-sleep
git add com.docker.driver.amd64-linux/on-sleep
git commit -m "disable freeze workaround"
```

**Comment by stewart123579** [comment]
Sadly the recurrence of **both** this issue and #1175 means that I've stayed with v1.13.0!  ☹️ 

It seems stable and hasn't caused me any problems.

**Comment by gitfoxi** [comment]
Hi. I'm a new Docker user. This is all a bit confusing. But me too!

```
Version 17.06.1-ce-mac24 (18950)
Channel: stable
54dc09c3e3
```

I am running the High Sierra beta:

```
10.13 Beta (17A360a)
```

```
sudo pmset -g
Enter PIN for 'Certificate For PIV Authentication (Yubico PIV Authentication)':
System-wide power settings:
Currently in use:
 standbydelay         10800
 standby              1
 halfdim              1
 hibernatefile        /var/vm/sleepimage
 powernap             1
 gpuswitch            2
 disksleep            10
 sleep                0
 autopoweroffdelay    0
 hibernatemode        3
 autopoweroff         0
 ttyskeepawake        1
 displaysleep         0
 acwake               0
 lidwake              1
```

I haven't tried any of the above workaround disablers yet. Seems like black magic. But that's my next step.

**Comment by lambdaq** [comment]
Same problem on Yosemite 

Version 17.12.0-ce-mac55 (23011)


```
Active Profiles:
Battery Power  		-1*
AC Power       		-1
Currently in use:
 standbydelay         10800
 standby              1
 halfdim              1
 hibernatefile        /var/vm/sleepimage
 darkwakes            0
 disksleep            10
 sleep                1
 autopoweroffdelay    14400
 hibernatemode        0
 autopoweroff         1
 ttyskeepawake        1
 displaysleep         2
 acwake               0
 lidwake              1
```

**Comment by esseti** [comment]
I still have the same problem, running Mojave, version Version 2.0.0.0-mac81 (29211)

everytime when mac wake up the docker shows the screen that allows you to reset it (but deletes everything) or exit. that's super annoying. 

Any help?

PS: there's no `no proxy` parameter in config. any idea what could be?
